### DISCUSSION TITLE
Seperate discussion pins per category

### DISCUSSION THREAD

**Discussion by Glazelf** [community/community#2856] [Discussions] [answered]
I would like to have seperate pins for each of my categories. 
I feel like it's quite messy to have pins from all categories combined and also makes the 4 pin limit more restraining.

Ideally there would be a seperate "global pin" feature that only show up on the "View All" tab, and then pins for each specific category.

**Comment by Griatch** [comment]
I agree with this, had this not already been here I'd have added this suggestion myself.

**Comment by imjohnbo** [comment, marked as answer]
Hi @Glazelf, category specific pins are now [available](https://github.blog/changelog/2022-11-14-discussions-category-specific-pins-and-enhancements/), let us know what you think. 👋

**Reply by Glazelf** [reply]
Haha I forgot about this post but after checking it out it looks clean.  
Mabye global pins could be pinned to the top in the "View All Discussions" tab the same way category pins are currently pinned, or category pins could show up in the banner where global pins now permanently show up.  
I'm just spitballing some ideas, these are not direct requests haha. Thanks for the heads up!
